src/utils/fetch.ts

```diff
@@ -143,3 +143,15 @@ export async function put<T>(
 ): Promise<T> {
   return send<T>('PUT', path, body, headers, props, accessToken);
 }
+
+export function isUrlEncoded(str: string) {
```

Comment:
Why this function name is isUrlEncoded?

Reply:
because it checks whether or not a string has already been encoded